plugins/TagFix_Opening_Hours.py

```diff
@@ -38,28 +38,36 @@ def init(self, logger):
         self.errors[32501] = self.def_class(item = 3250, level = 3, tags = ['value', 'fix:chair'],
             title = T_('Invalid Opening Hours'))
 
-    def sanitize_tags(self, tags):
-        if 'opening_hours' not in tags:
+    # Function is also called from ConditionalRestrictions
+    def sanitize_openinghours(self, openinghours_value):
+        if not module_PyKOpeningHours:
             return
-
         parser = OpeningHours()
-        parser.setExpression(tags['opening_hours'])
+        parser.setExpression(openinghours_value)
         if parser.error() == Error.SyntaxError or parser.error() == Error.IncompatibleMode:
-            return {"class": 32501, "subclass": 1, 'text': {'en': 'The opening_hours value is invalid and could not be parsed'}}
+            return {"isValid": False}
         sanitized_field = parser.normalizedExpression()
         # Ignore trivial changes that can be fixed by bots rather than humans like spaces, case errors, etc
         simplify = lambda s: s.replace(' ', '').replace('24:00', '00:00').replace('0', '').lower()
```

Comment:
I would be OK with the .lower() being removed, this is from before my time.

It would however create more work for people applying fixes suggested by osmose, obviously.

Reply:
I'd prefer to remove `lower()` too, as in practice I'm not aware of bots fixing case-errors (pretty error-prone I think, since anything can be in the fallback rule, like `9:00-18:00 || "by appointment made on monday"` shouldn't become `9:00-18:00 || "by appointment made on Mo"`, so any bot-fix should be very careful.
@frodrigo ?

Reply:
Right. My semi-interactive "bot fixer" actually uses KOpeningHours for parsing and fixing ;)

Reply:
Haha oops, ok, in that case the question is indeed only if we should add case-errors to the UI or not

Reply:
Currently there's no error given if you use `mo` or `JAN` rather than Mo or Jan. Would we want to enable this, so they become visible in the UI?

Reply:
Yes we can.
The initial idea was to avoid boring contributor with obvious fix than parser are totally aware of.

Reply:
Implemented in #1431